FIRMWARE/PocketSDVX/PocketSDVX.ino

```diff
@@ -42,6 +42,16 @@ int currMode = EEPROM.read(0);
 #define FX_R_LED  21
 #define BT_ST_LED 16
 
+// Define modes
+#define MODE_JOYSTICK 1
+#define MODE_KEYBOARD 2
+#define MODE_LIGHT_ON 1
+#define MODE_LIGHT_OFF 0
+
+// Eeprom addresses
+#define EEPROM_CONTROL_ADDRESS 0
+#define EEPROM_LIGHT_ADDRESS 1
+
```

Comment:
Define new variables for later autocomplete. No more "Which one is joystick mode, which one is keyboard mode"